targets/targets.json

```diff
@@ -7561,6 +7986,8 @@
             "CYBSP_WIFI_CAPABLE"
         ],
         "device_name": "CY8C6245LQI-S3D72",
+        "mbed_rom_start": "0x10000000",
+        "mbed_rom_size": "0x80000",
         "mbed_ram_start": "0x08002000",
         "mbed_ram_size": "0x0003D800",
```

Comment:
Are these RAM values correct? They seem to have similar issues to the `CY8C624ABZI-D44` issues.